color/color.go

```diff
@@ -0,0 +1,78 @@
+package multicolor
+
+import (
+	"fmt"
+	"strings"
+	"unicode"
+
+	"github.com/ghishadow/color"
+)
+
+// New returns a color attribute build from supplied attribute names.
+// This function will panic if invalid attributes are supplied.
+func New(attributes ...interface{}) *ColorWriter {
```

Comment:
This should go in the writer.go file

Reply:
Why only this one? In fact, I separated the writter functionnaly from the color object.